packages/web/README.md

```diff
@@ -45,3 +45,7 @@ On completion, two local html files containing visual bundle trees for client an
 ### Frontier
 
 Frontier mode is managed by the `NEXT_PUBLIC_IS_FRONTIER=true` env var, deployed from `master` branch.
+
+## Environment Variables
+
```

Comment:
Could we add one sentence of how the config works? Perhaps replace everything in this section with:

> By default, chain configuration is hardcoded and determined by the `NEXT_PUBLIC_IS_TESTNET` env var for developer convenience and simplicity. Please directly change those values should the config be changed from here on. For temporary overrides, consult the [.env](.env) file.